### PR TITLE
[interfaces] Use "Vlan"+tag instead of name for linux vlan device names

### PR DESCRIPTION
**Pull request by taoyl-ms**
See #1003 for details.

### CODE REVIEW

**Comment by jleveque** [comment]
It appears this commit will change the way we name Vlan interfaces? If so, this will also impact the DHCP relay agent.

File: files/image_config/interfaces/interfaces.j2 [hunk 1]
```diff
@@ -62,7 +62,7 @@ iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 allow-hotplug {{ member }}
 iface {{ member }} inet manual
     pre-up ifconfig {{ member }} up mtu 9100
-    post-up brctl addif {{ vlan }} {{ member }} || true
+    post-up brctl addif Vlan{{ VLAN[vlan]['vlanid'] }} {{ member }} || true
```
Comment (stcheng) [inline files/image_config/interfaces/interfaces.j2]: how do we assign a VLAN ID if the VLAN name is not a number?

**Comment by lguohan** [comment]
@taoyl-ms, looks like you need to grab other places which use the 'name' to the new format.

**Comment by lguohan** [comment]
@taoyl-ms , can you revive this PR?

**Comment by taoyl-ms** [comment]
Issue solved by a different approach in https://github.com/Azure/sonic-buildimage/pull/695.

**Pull request closed without merging**